### DISCUSSION TITLE
Schedule: On Specific Weekday Once Every Second Month

### DISCUSSION THREAD

**Discussion by karlhorky** [renovatebot/renovate#10685] [Request Help]
I am trying to schedule Renovate to run on every first Thursday of every **second** month, and I have the following - does Renovate support this?

```
on Thursday on the first day instance of January, March, May, July, September, November after 4pm
```

Seems overly complicated, but if I make it simpler, it gives me weird errors like starting in 100 years or something 😅

I tested it on repl.it, and this seems like it is this the correct Later.js schedule... https://replit.com/@karlhorky/SupportiveLivelySorting#index.js

```
[
  2021-07-01T08:58:36.652Z,
  2021-09-02T00:00:00.000Z,
  2021-11-04T00:00:00.000Z,
  2022-01-06T00:00:00.000Z,
  2022-03-03T00:00:00.000Z,
  2022-05-05T00:00:00.000Z,
  2022-07-07T00:00:00.000Z,
  2022-09-01T00:00:00.000Z,
  2022-11-03T00:00:00.000Z,
  2023-01-05T00:00:00.000Z
]
```

The parsed schedule in Renovate (copied it from the dashboard) seems to also match (also added it in the repl above):

```js
const renovateSchedule = {
  "parsedSchedule": {
    "schedules": [
      {
        "d": [
          5
        ],
        "dc": [
          1
        ],
        "M": [
          1,
          3,
          5,
          7,
          9,
          11
        ],
        "t_a": [
          57600
        ]
      }
    ],
    "exceptions": [],
    "error": -1
  }
}

console.log(later.schedule(renovateSchedule.parsedSchedule).next(1, new Date('2021-07-01')))
console.log(later.schedule(renovateSchedule.parsedSchedule).next(1, new Date('2021-08-01')))
console.log(later.schedule(renovateSchedule.parsedSchedule).next(1, new Date('2021-10-01')))

// 2021-07-01T16:00:00.000Z
// 2021-09-02T16:00:00.000Z
// 2021-11-04T16:00:00.000Z
```

cc @fullstackzach from https://github.com/renovatebot/config-help/issues/953

cc @fgblomqvist from https://github.com/renovatebot/renovate/issues/9805 and subsequent PR #9819

cc @cfra from https://github.com/renovatebot/renovate/issues/4831 and subsequent PR #4832

**Comment by karlhorky** [comment]
@rarkins if this is supported, would a PR adding this and other examples be added to the documentation be accepted?

The docs for Later.js are somewhat lacking, and if this was at least in the Renovate docs then I think that would save a lot of people a lot of time.

**Comment by fgblomqvist** [comment]
I don't know a way to make it shorter unfortunately, and like you said, their (later.js) docs are lacking. I believe the goal is to replace later.js with just a Cron parser though, so not sure if there's a point in expanding the renovate docs 🤷‍♂️
